guides/common/modules/proc_setting-up-pull-client.adoc

```diff
@@ -0,0 +1,34 @@
+[id="setting-up-pull-client_{context}"]
+= Setting Up the Remote Execution Pull Client
+
+For {SmartProxies} in `pull-mqtt` mode, hosts can subscribe to remote jobs using the remote execution pull client.
+No SSH connection is required.
+
+Use this procedure to configure a host to use the pull client:
+
+.Prerequisites
+* The host is registered to {ProjectName}.
+* The host's {SmartProxy} is in `pull-mqtt` mode.
+* The {Team} {project-client-name} repository is enabled and synchronized on {ProjectServer}, and enabled on the host.
+* The host is able to communicate with its {SmartProxy} over MQTT using port 1883.
+* The host is able to communicate with its {SmartProxy} over HTTPS.
+
+.Procedure
+. On the host, install the `katello-pull-transport-migrate` RPM package:
++
+----
+# yum install katello-pull-transport-migrate
```

Comment:
This seems to depend on the OS/EL version. what about el8? debian? Is this EL only?

Reply:
`subscription-manager` is required, so I'd say yes it's EL-only.

Reply:
FYI: You can also run subscription-manager on sles and Debian/Ubuntu: https://apt.atix.de/

So maybe make the "yum" conditional?

Reply:
@adamruzicka @ehelms Is the pull client supported on Debian?

Reply:
I'd be inclined to say no. The binaries we build are static and should run essentially anywhere as long as the platform and architecture match, but afaik we don't build packages for debian.

Reply:
Thanks for the info. I guess the suggestion remains to split the "yum install" command into EL7 and EL8/9 (dnf).